<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uicomchatpage</class>
 <widget class="QWidget" name="uicomchatpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTextEdit" name="sendMsgBlock">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>871</width>
     <height>161</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);
font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QScrollArea" name="recvMsgBlock">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>501</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="recvMsgBlockContent">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>852</width>
      <height>499</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="sendMsgButton">
   <property name="geometry">
    <rect>
     <x>805</x>
     <y>675</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);
border-radius : 8px;</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
